adapter/internal/oasparser/envoyconf/access_loggers.go

```diff
@@ -108,6 +108,7 @@ func getGRPCAccessLogConfigs(conf *config.Config) *config_access_logv3.AccessLog
 				Timeout: ptypes.DurationProto(conf.Analytics.Adapter.GRPCRequestTimeout),
 			},
 		},
+		AdditionalRequestHeadersToLog: []string{"x-original-gw-url"},
```

Comment:
Shall we have a constant here too?

Reply:
remove this

Reply:
Fixed with: be7bc20383c53e5ecb044bd929e2d385420e82a0